packages/backend/models/albumSchema.js

```diff
@@ -9,16 +9,6 @@ const AlbumSchema = new mongoose.Schema({
   track_list: [{ type: String }],
   album_cover: String,
   spotify_link: String,
-  // maybe to hold the id of the reviews?
```

Comment:
would removing the array of reviews specific to the album be a problem or is it just better styling? i think we do have an albums id inside of reviews so i guess it would be better to just keep it inside one, but i'm curious about what yall think